code/modules/food/drinks/flask.dm

```diff
@@ -0,0 +1,33 @@
+/obj/item/reagent_containers/food/drinks/flask
+	name = "\improper Facility Director's flask"
```

Comment:
why is the baseline type the directors flask?

Reply:
its a copypaste not an add i think